webtau-feature-testing/test-expectations/scenarios/concept/runOnlySelected/run-details.json

```diff
@@ -1,15 +1,19 @@
 {
   "scenarioDetails" : [ {
     "scenario" : "step one",
+    "shortContainerId" : "runOnlySelected.groovy",
```

Comment:
I understand the purpose and how it tested your scenario, but I don't like having them in every expectation very much. Don't have an alternative yet.

Reply:
Tell me about it, I didn't really want to have to modify all these files! :)

One alternative would be to allow feature test to pass an optional set of fields to capture from the `StandaloneTest` in addition to the default scenario and stepsSummary.